src/main/java/seedu/address/logic/parser/StageManager.java

```diff
@@ -1,14 +1,30 @@
 package seedu.address.logic.parser;
 
+import static java.util.Objects.requireNonNull;
+
+import javafx.stage.Stage;
 import seedu.address.model.course.Course;
 
 /**
  * This class manages the stages of the application.
```

Comment:
I thinks the Javadoc format should be "Manages the stages of the application." instead